src/o.js

```diff
@@ -0,0 +1,31 @@
+var _curry2 = require('./internal/_curry2');
+
+
+/**
+ * `o` is a binary, curried composition function that returns a unary function.
+ * Like `compose`, `o` performs right-to-left function composition. Unlike `compose`,
+ * the rightmost function passed to `o` will be invoked with only one argument.
+ *
+ * @func
+ * @memberOf R
+ * @category Function
+ * @sig (b -> c) -> (a -> b) -> (a -> c)
+ * @param {Function} f
+ * @param {Function} g
+ * @return {Function}
+ * @see R.compose, R.pipe
+ * @example
+ *
+ *      var classyGreeting = name => "The name's " + name.last + ", " + name.first + " " + lastName
+ *      var yellGreeting = R.o(R.toUpper, classyGreeting);
+ *      yellGreeting({first: 'James', last: 'Bond'}); //=> "THE NAME'S BOND, JAMES BOND"
+ *
+ *      R.o(Math.abs, R.multiply(2))(-4) //=> 8
```

Comment:
You should choose an example which demonstrates the order in which the functions are applied.